tools/deb/envoy_bootstrap_v2.json

```diff
@@ -2,21 +2,22 @@
   "node": {
```

Comment:
I found it difficult to edit this file due to the inconsistent indentation scheme. As a result, I ended up reformatting this file so that indentation was consistent and also made it proper JSON by removing trailing commas.